source/screens/caseScreens/CaseOverview.tsx

```diff
@@ -380,6 +382,17 @@ function CaseOverview(props): JSX.Element {
   const activeCases = getCasesByStatuses([NOT_STARTED, ACTIVE]);
   const closedCases = getCasesByStatuses([CLOSED]);
 
+  const newApplicationCaseIndex = activeCases.findIndex(
+    ({ status }) => status?.type === NOT_STARTED_NEW_APPLICATION_VIVA
+  );
+  const newApplicationCase =
+    newApplicationCaseIndex >= 0
+      ? activeCases.splice(newApplicationCaseIndex, 1)[0]
+      : undefined;
+
+  const showActiveCases = activeCases.length > 0 && !newApplicationCase;
+  const showClosedCases = closedCases.length > 0 && !newApplicationCase;
```

Comment:
Probably doesn't hurt to show active/closed cases regardless if there's a "new" case (if we imagine in the future where more than EKB exists).